src/modules/fancyzones/lib/JsonHelpers.cpp

```diff
@@ -21,6 +21,12 @@ namespace
     constexpr int c_blankCustomModelId = 0xFFFA;
 
     const wchar_t* FANCY_ZONES_DATA_FILE = L"zones-settings.json";
+
+    std::wstring ExtractVirtualDesktopId(const std::wstring& deviceId)
+    {
+        // Format: <device-id>_<resolution>_<virtual-desktop-id>
```

Comment:
Why should data has to be stored in this format? Much easier to save device-id, resolution and virtual-desktop-id separately. And if virtual-desktop-id was used as key in `deviceInfoMap`, adding and removing process also could be easier and faster.

Reply:
I don't know why do we need resolution here, but device id and virtual desktop id are bot necessary. Key in `deviceInfoMap` can't be only virtual desktop id, as You need both of them to uniquely define work area.

Reply:
We used screen resolution in the FZ settings when they were saved in the registry but it has been removed in the new json format. So I guess we don't need the resolution for the virtual desktopId either. Please check with Stefan if it's OK to not use it.